hs_access_control/models/resource.py

```diff
@@ -127,46 +200,159 @@ def owners(self):
 
         This is a property so that it is a workalike for a prior explicit list.
 
+        Groups and communities cannot own resources.
+
         For immutable resources, owners are not modified, but do not have CHANGE privilege.
         """
         return User.objects.filter(is_active=True,
                                    u2urp__privilege=PrivilegeCodes.OWNER,
                                    u2urp__resource=self.resource)
 
-    def get_users_with_explicit_access(self, this_privilege, include_user_granted_access=True,
-                                       include_group_granted_access=True):
+    def get_users_with_explicit_access(self, this_privilege,
```

Comment:
I am assuming this function works. May be there is way to break this function to multiple sub functions.

Reply:
The best I could do is to break the clauses into subfunctions.